docs/examples/evaluation/prometheus_evaluation.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>Re: 1. It would be great if possible to provide a similar cost breakdown for GPT-4. We performed the same evaluations using GPT-4 as Prometheus right?</p><p> </p><p>Re: 2. That's interesting. Are we saying that GPT-4 feedback is always right? If not, then I'd say that "both" evaluators can get the feedback wrong and should be used cautiously. (In the paper, human evaluators actually gave Prometheus feedback higher ratings, like 58% of the time)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
I think I will probably do manual comparisions on the feedbacks and see if GPT-4 feedbacks are always correct.